### DISCUSSION TITLE
Suggested time entries from longer period

### DISCUSSION THREAD

**Discussion by kimmo-ojanpera** [mcndt/obsidian-toggl-integration#102] [Feature Requests]
Hi,

I assume that when I create a time entry, I'm given suggestions from the week. Could the time period be a bit longer? Now when I have work that repeats weekly, I have to write the entries manually.

BR

Kimmo

**Comment by mcndt** [comment]
Hi Kimmo, you are right in that it fetches the entries from the past 7 days. For performance reasons I have kept it this way (the Toggl servers start responding quite slowly for longer time ranges, making the plugin feel sluggish).

However I appreciate the use case for weekly recurring entries. Perhaps 8 or 9 days would satisfy in this case?

**Reply by kimmo-ojanpera** [reply]
> Perhaps 8 or 9 days would satisfy in this case?

I assume that it would work really well.

Thanks for the plugin!

Kimmo

**Reply by mcndt** [reply]
I changed it to up to 10 days in the past in version 0.9.0. Will be available very shortly.
